kolibri/plugins/device/assets/src/views/ManageContentPage/ChannelPanel/WithImportDetails.vue

```diff
@@ -232,6 +232,8 @@
   .col-3 {
     display: flex;
     align-items: center;
+    justify-content: flex-end;
```

Comment:
FWIW I did this: https://github.com/learningequality/kolibri/pull/6507 - I found the issue and fixed it and didn't see the open issue assigned to @jonboiser by adding `flex-grow: 1` to the sibling element.

Reply:
Just flagging for posterity that usually we don't "allow" flexbox in Kolibri, but we made a decision to leave it here because we were in a rush for 0.13 and @jonboiser found it faster to build it this way than using `KGrid`, which is inline-block-based. I've heard similar feedback that flexbox is faster and simpler to use than `KGrid` from at least two other devs.

I've been planning on trying to re-implement this responsive UI using `KGrid` as an experiment just to compare the diff.